### PR TITLE
Search profile

### PR DESCRIPTION
**Pull request by BelideNikhil**
### Features

   + Search with Debounce
   + Sort videos by date
   + User profile page

### CODE REVIEW

**Review by Shrinkhlarajpoot — APPROVED** [review]

File: src/Utils/debounceFunction.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export function debounce(callback, delay = 400) {
+    let timer;
+    return function (...args) {
+        clearTimeout(timer);
+        timer = setTimeout(() => {
+            callback(...args);
+        }, delay);
+    };
+}
```
Comment (pritam-kr) [inline src/Utils/debounceFunction.js:9]: I love to know about this please explain. some points I didn't understand

**Comment by pritam-kr** [comment]
Every Things looks good to me

**Review by pritam-kr — APPROVED** [review]

File: src/Utils/debounceFunction.js (hunk 1 above)
Comment (BelideNikhil) [reply src/Utils/debounceFunction.js:9]: sure!

**Review by msdeshmukh009 — APPROVED** [review]
The code looks clean. The search feature is working really nicely.

**Pull request merged**